stubs/python-jose/jose/backends/base.pyi

```diff
@@ -0,0 +1,19 @@
+from typing import Any
+
+from ..utils import base64url_encode as base64url_encode, ensure_binary as ensure_binary
```

Comment:
This is the first instance, but there are more below. This is probably a stubgen artifact, but we should not re-export, except from `__init__.pyi`. In this particular case, we can just completely remove the import.

Reply:
Correct, I just glazed over the imports without thinking about it. Removed